app/assets/javascripts/discourse/app/components/user-preferences/user-passkeys.hbs

```diff
@@ -0,0 +1,59 @@
+<div class="control-group pref-passkeys">
```

Comment:
did we get bored of gjs at this point ? :p

Reply:
Nope but I'd have to make the `format-date` helper work within `gjs`, and that would expand the scope of the PR a bit too much for my comfort.

Reply:
You can do this:

```
formatDate = getOwner(this).resolveRegistration("helper:format-date");
```

and then `{{this.formatDate}}`

Reply:
Ah, nice, that worked, but the `gjs`ification failed at a later step that looks like a bigger problem. It failed with the `<UserPreferences::PasskeyOptionsDropdown` component next, which is a select-kit dropdown. The reason was: 

```
Uncaught Error: Assertion Failed: EmberObject.create no longer supports defining computed properties. Define computed properties using extend() or reopen() before calling create().
```

The content of the SK dropdown uses `computed()`, so this seems like a harder blocker? 

I tried switching to a `FloatKit` menu, but that one would be missing the consistent SK dropdown menu styling, it would look like this (without any styling): 

<img width="711" alt="image" src="https://github.com/discourse/discourse/assets/368961/d2d12bf5-2876-40e0-b5b6-d8864a86ee8d">

and again I hesitate to expand the scope of the PR.